react/Widget.tsx

```diff
@@ -9,6 +9,7 @@ interface Props {
   width: string
   height: string
   stars: string
+  tag: string
```

Comment:
This must be optional to avoid breaking changes:

```suggestion
  tag?: string
```